src/zenml/cli/utils.py

```diff
@@ -217,11 +220,48 @@ def print_stack_configuration(
 
     # capitalize entries in first column
     rich_table.columns[0]._cells = [
-        component.upper() for component in rich_table.columns[0]._cells  # type: ignore[union-attr]
+        component.upper()  # type: ignore[union-attr]
+        for component in rich_table.columns[0]._cells
     ]
     console.print(rich_table)
 
 
+def print_flavor_list(
+    flavors: List[FlavorWrapper],
+    component_type: StackComponentType,
+) -> None:
+    """Prints the list of flavors."""
+    from zenml.integrations.registry import integration_registry
+
+    flavor_table = []
+    for f in flavors:
+        reachable = False
+
+        if f.integration:
+            if f.integration == "built-in":
+                reachable = True
+            else:
+                reachable = integration_registry.is_installed(f.integration)
+
+        else:
+            try:
+                validate_flavor_source(f.source, component_type=component_type)
+                reachable = True
+            except (AssertionError, ModuleNotFoundError, ImportError):
+                pass
+
+        flavor_table.append(
+            {
+                "FLAVOR": f.name,
+                "INTEGRATION": f.integration,
+                "READY-TO-USE": ":white_check_mark:" if reachable else "",
```

Comment:
I feel this column is not self explanatory - could we add a description above or below the table that explains what it means to be READY-TO-USE and what I can do to make something ready

Reply:
Nice suggestion! I have added a nice warning message underneath the table.